python/cudf/cudf/core/dataframe.py

```diff
@@ -3785,6 +3785,13 @@ def to_csv(
             chunksize,
         )
 
+    def repeat(self, repeats, axis=None):
+        assert axis in (None, 0)
+        cols = libcudf.filling.repeat(self._columns, repeats)
+        # to preserve col names, need to get it from old _cols dict
+        column_names = self._cols.keys()
+        return DataFrame(dict(zip(column_names, cols)))
```

Comment:
This doesn't look like it's repeating the index, we need to handle the index column(s) as well.